Runtime/Analytics/AmplitudeRuntimeLogger.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+
+namespace ReadyPlayerMe.Core.Analytics
+{
+    /// <summary>
+    /// Runtime analytics for sample which runs in the editor only.
+    /// </summary>
+    public class AmplitudeRuntimeLogger : IAnalyticsRuntimeLogger
+    {
+        private const string RUN_QUICKSTART_SCENE = "run quickstart scene";
+        private const string PERSONAL_AVATAR_LOADING = "personal avatar loading";
+        private const string AVATAR_URL = "avatar url";
+        private const string PERSONAL_AVATAR_LOAD_BUTTON_CLICKED = "personal avatar load button clicked";
+
+        public void LogRunQuickStartScene()
+        {
+            LogEvent(RUN_QUICKSTART_SCENE);
+        }
+
+        public void LogPersonalAvatarLoading(string url)
+        {
+            LogEvent(PERSONAL_AVATAR_LOADING, new Dictionary<string, object>
+            {
+                { AVATAR_URL, url }
+            });
+        }
+
+        public void LogLoadPersonalAvatarButton()
+        {
+            LogEvent(PERSONAL_AVATAR_LOAD_BUTTON_CLICKED);
+        }
+
+        private void LogEvent(string eventName, Dictionary<string, object> eventProperties = null, Dictionary<string, object> userProperties = null)
+        {
+            if (!CoreSettingsHandler.CoreSettings.EnableAnalytics) return;
```

Comment:
we could potentially move this check inside the AmplitudeEventLogger.LogEvent as we do the same if return statement in the Editor logger. To remove a tiny amount of code duplication.

Reply:
I agree it could remove duplication, but this will make the AmplitudeEventLogger script dependent on CoreSettings. This script is currently decoupled and standalone with very few dependency.